### PR TITLE
Transfer to new drone

### PR DESCRIPTION
**Pull request by tasharnvb**
_~156~ 184 builds later..._

I will rebase if this PR is accepted.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ RUN apk upgrade --no-cache && \
 
 COPY bin/run.sh /run.sh
 COPY conf.d /etc/nginx/conf.d
+COPY tests/nginx.conf /nginx.conf
```
Comment (vaijab) [inline Dockerfile]: Does this mean that test config will be included in a final image? If that's the case, you want a separate Dockerfile for testing or a way to enable copying the test config only at test time.

**Review by vaijab — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (tasharnvb) [reply Dockerfile]: It does, I didn't think about that. I will change this.

Comment (jits) [reply Dockerfile]: You could consider using a [multi-stage docker build](https://docs.docker.com/engine/userguide/eng-image/multistage-build/)

**Comment by tasharnvb** [comment]
Note: I could not find a way to only pass the test config to the container at runtime (what is suggested in the repo [here](https://github.com/UKHomeOffice/docker-nginx#config-file-via-env-variable) does not work in new drone) so instead a new image is made with the config already built into it.

**Review by vaijab — APPROVED** [review]

**Pull request merged**